x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/audit/logfile/LoggingAuditTrailTests.java

```diff
@@ -627,21 +627,23 @@ public void testSecurityConfigChangeEventFormattingForRoles() throws IOException
         CapturingLogger.output(logger.getName(), Level.INFO).clear();
 
         final String keyId = randomAlphaOfLength(10);
+        final TimeValue newExpiration = ApiKeyTests.randomFutureExpirationTime();
```

Comment:
Nit: I'd randomize between an actual value and `null`, i.e., `final TimeValue newExpiration = randomFrom(ApiKeyTests.randomFutureExpirationTime(), null);` and tweak the assertion further down.

Reply:
Yes, I agree.